manifests/init.pp

```diff
@@ -37,10 +37,15 @@
 #
 class jmxtrans (
   Optional[String[1]] $package_name = undef,
+  Optional[String[1]] $package_version = undef,
   Optional[String[1]] $service_name = undef,
   Optional[String[1]] $package_source = undef,
   Optional[String[1]] $package_provider = undef,
+  Optional[String[1]] $working_directory = undef,
   Boolean $manage_service_file = false,
+  Optional[Array[String]] $systemd_environment_vars = [],
+  String[1] $systemd_template = 'jmxtrans/jmxtrans.service.epp',
+  String[1] $binary_path = '/usr/share/jmxtrans/bin/jmxtrans.sh',
```

Comment:
This should be `'/usr/share/jmxtrans/bin/jmxtrans'` so as not to change the current behavior.